### ISSUE TITLE
Dynamic change past and future option

### ISSUE THREAD

**Issue by emil-zhivkov** [puranjayjain/md-date-time-picker#247]
Hello can i dynamic change future or past option. Or is there a possibility to use this date picker like date range picker ?

**Comment by puranjayjain** [comment]
@emil-zhivkov  Sorry, but we don't support that functionality out of the box yet.

**Comment by emil-zhivkov** [comment]
Can i change future or past options dinamicly ?

**Comment by puranjayjain** [comment]
@emil-zhivkov yes you can do that see https://puranjayjain.github.io/md-date-time-picker/#documentation/past

**Comment by emil-zhivkov** [comment]
@puranjayjain tell me how i have start date and end date 2 datepickrs how i limit it start date always be <= end date i need dynamicly change past and future options

**Comment by puranjayjain** [comment]
@emil-zhivkov sorry I didn't understand your question, can you rephrase it?

**Comment by emil-zhivkov** [comment]
OKay this situation:
i have 2 date pickers 1. Start date and 2. End date i must choose period.
and when i select some date from 1. Start date u must dynamicly set past date to 2. End date :)
or when u choose 2 u myst dynamicaly set limit tu future option for 1.
How can i do that ?
